ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/event/solicitor/page/SolSummary.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.divorce.ccd.event.solicitor.page;
+
+import uk.gov.hmcts.reform.divorce.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.reform.divorce.ccd.builder.FieldCollectionBuilderExtension;
+import uk.gov.hmcts.reform.divorce.ccd.model.CaseData;
+
+public class SolSummary implements CcdPageConfiguration {
+
+    @Override
+    public void addTo(final FieldCollectionBuilderExtension fieldCollectionBuilder) {
+
+        fieldCollectionBuilder
+            .page("SolSummary")
+            .mandatoryNoSummary(CaseData::getD8ReasonForDivorce, "D8StatementOfTruth=\"Banana\"")
+            .label("LabelSolAppSummaryPara-1", "# Before you submit")
+            .label("LabelSolAppSummaryPara-2", "## What happens next")
+            .label(
+                "LabelSolAppSummaryPara-3",
+                "### Please continue to submit your application on the next screen. \r\nThe application "
+                    + "will be checked. If it’s correct, you’ll be sent a notice of issue. The respondent will "
+                    + "also receive a copy of the application unless you have chosen to personally effect service.")
```

Comment:
should we define these labels in properties file and load it in `CcdPageConfiguration` and here you just do 
```
.label(labelProps.get("appSummaryPara"));
```

Reply:
I am not so sure, as it will be another set of files to view the content of the page.  We can discus and implement later if we want to go down this route.